README.md

```
@@ -19,26 +19,11 @@ O nome das actions é composto pelos seguintes componentes (entre [ ])
-git checkout production
-git pull     
-git log -n 1 --pretty=format:"%h" | tail -n 1                 # mostra o ultimo commit
-git tag -l                                                    # lista as tags atuais
-git tag -a v1.1.0 -m "Aegro workflows - add k8s actions"      # criar a tag com um numero superior (semantic version)
-git push origin production --tags                             # Enviar a tag     
-```
-
-[**É importante que sejam criadas as releases informando o que foi alterado na versão.**](https://docs.github.com/pt/repositories/releasing-projects-on-github/managing-releases-in-a-repository)
+O corte da tag e da release é automático e a propagação do bump para os consumidores também. O fluxo completo, de ponta a ponta, está em [`docs/release-automation.md`](docs/release-automation.md).
 
 ## Contribuindo
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)